packages/main/src/TextArea.hbs

```diff
@@ -25,6 +28,7 @@
 		?required="{{required}}"
 		aria-required="{{required}}"
 		aria-label="{{ariaLabelText}}"
+		aria-labelledby="{{_id}}-acc-text"
```

Comment:
not sure if by adding aria-labelledby would prevent aria-label from reading, can you sync that with an expert, if not already discussed of course

Reply:
This is not a problem. The aria-label attribute is still read out by the screen reader